### PR TITLE
PersonaBar, Settings, Servers, System Info, Web subtab - Adds the .NET Framework Product Version

### LINKED ISSUE #6155: [Enhancement]: PersonaBar, Settings, Servers, System Info, Web subtab that adds the .NET Framework Product Version
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of problem

On the Server Info panel under Web, only the .NET Framework CLR version is visible.

### Description of solution

After recently adding the enhanced NETFrameworkVersion in #6143, seemed like a reasonable idea to also add it to the PersonaBar details panel for the Web info. And also better label the CLR vs Product versions (including tooltip).
 
My proposed solution looks like this:
image

### Description of alternatives considered

_No response_

### Anything else?

This is not a very exciting enhancement, but was a good learning experience in figuring out how to work on PersonaBar stuff. Huge thanks to @valadas for re-teaching me how to build the client side (React) components!

### Do you be plan to contribute code for this enhancement?

- [X] Yes

### Would you be interested in sponsoring this enhancement?

- [ ] Yes

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by jeremy-farrance**
Fixes #6155 

## PersonaBar, Settings, Servers, System Info, Web subtab that adds the .NET Framework Product Version

This screenshot shows what these changes will look like:
image

### CODE REVIEW

File: Dnn.AdminExperience/ClientSide/Roles.Web/src/components/roles/RoleEditor/index.jsx [hunk 1]
```diff
@@ -194,6 +194,11 @@ class RolesEditor extends Component {
     render() {
         let { state, props } = this;
         const columnOne = <div key="editor-container-columnOne" className="editor-container">
+            <div className="editor-row divider" style={{marginTop:"-3rem"}}>
+                <Label
+                    label={resx.get("RoleId") + ": " + state.roleDetails.id}
+                    style={{ fontWeight: "bolder" }} />
+            </div>
```
Comment (valadas) [inline Dnn.AdminExperience/ClientSide/Roles.Web/src/components/roles/RoleEditor/index.jsx:201]: Looks like this comes from your other PR

File: Dnn.AdminExperience/ClientSide/Roles.Web/src/components/roles/RoleRow/index.jsx [hunk 2]
```diff
@@ -119,7 +119,9 @@ RoleRow.propTypes = {
     openId: PropTypes.string,
     currentIndex: PropTypes.number,
     visible: PropTypes.bool,
-    roleIsApproved: PropTypes.bool
+    roleIsApproved: PropTypes.bool,
+    addIsClosed: PropTypes.bool,
+    children: PropTypes.node
```
Comment (valadas) [inline Dnn.AdminExperience/ClientSide/Roles.Web/src/components/roles/RoleRow/index.jsx:124]: Looks like this comes from your other PR

File: Dnn.AdminExperience/ClientSide/Roles.Web/src/components/roles/UsersInRole/UserRow/index.jsx [hunk 3]
```diff
@@ -138,7 +138,9 @@ UserRow.propTypes = {
     userDetails: PropTypes.object.isRequired,
     index: PropTypes.number,
     saveUser: PropTypes.func.isRequired,
-    deleteUser: PropTypes.func.isRequired
+    deleteUser: PropTypes.func.isRequired,
+    roleName: PropTypes.string.isRequired,
+    dispatch: PropTypes.func.isRequired
```
Comment (valadas) [inline Dnn.AdminExperience/ClientSide/Roles.Web/src/components/roles/UsersInRole/UserRow/index.jsx:143]: Looks like this comes from your other PR

File: Dnn.AdminExperience/ClientSide/Servers.Web/src/services/smtpServerService.js [hunk 4]
```diff
@@ -17,7 +17,7 @@ const sendTestEmail = function (parameters) {
     return serviceFramework.post(getControllerName(), "SendTestEmail", parameters);
 };
 
-const getOAuthProviders = function(){
+const getOAuthProviders = function () {
```
Comment (valadas) [inline Dnn.AdminExperience/ClientSide/Servers.Web/src/services/smtpServerService.js:20]: This appears to be unrelated

```suggestion
const getOAuthProviders = function(){
```

File: Dnn.AdminExperience/Dnn.PersonaBar.Extensions/admin/personaBar/Dnn.Roles/App_LocalResources/Roles.resx [hunk 5]
```diff
@@ -189,6 +189,9 @@
   <data name="RoleName.Text" xml:space="preserve">
     <value>Role Name</value>
   </data>
+  <data name="RoleId.Text" xml:space="preserve">
+    <value>Role ID</value>
+  </data>
```
Comment (valadas) [inline Dnn.AdminExperience/Dnn.PersonaBar.Extensions/admin/personaBar/Dnn.Roles/App_LocalResources/Roles.resx:194]: Looks like this comes from your other PR

File: Dnn.AdminExperience/Dnn.PersonaBar.Extensions/admin/personaBar/Dnn.SiteImportExport/scripts/bundles/siteimportexport-bundle.js.LICENSE.txt [hunk 6]
```diff
@@ -0,0 +1,5 @@
+/*!
+	Copyright (c) 2018 Jed Watson.
+	Licensed under the MIT License (MIT), see
+	http://jedwatson.github.io/classnames
+*/
```
Comment (valadas) [inline Dnn.AdminExperience/Dnn.PersonaBar.Extensions/admin/personaBar/Dnn.SiteImportExport/scripts/bundles/siteimportexport-bundle.js.LICENSE.txt:5]: Unrelated but probably auto-generated, so we could keep it.

**Review by valadas — APPROVED** [review]
Looks good overall with a minor unrelated change and there is probably here a commit that is for #6154 So @dnnsoftware/approvers we need to merge the other before this one and most of my concerns will be resolved.

File: Dnn.AdminExperience/ClientSide/Servers.Web/src/services/smtpServerService.js (hunk 4 above)
Comment (jeremy-farrance) [reply Dnn.AdminExperience/ClientSide/Servers.Web/src/services/smtpServerService.js:20]: Yes, unnecessary, but this eliminates a warning and makes the code consistent with the other 3 functions above.

File: Dnn.AdminExperience/ClientSide/Roles.Web/src/components/roles/RoleEditor/index.jsx (hunk 1 above)
Comment (jeremy-farrance) [reply Dnn.AdminExperience/ClientSide/Roles.Web/src/components/roles/RoleEditor/index.jsx:201]: Yes. Sorry.

File: Dnn.AdminExperience/Dnn.PersonaBar.Extensions/admin/personaBar/Dnn.SiteImportExport/scripts/bundles/siteimportexport-bundle.js.LICENSE.txt (hunk 6 above)
Comment (jeremy-farrance) [reply Dnn.AdminExperience/Dnn.PersonaBar.Extensions/admin/personaBar/Dnn.SiteImportExport/scripts/bundles/siteimportexport-bundle.js.LICENSE.txt:5]: I actually deleted this twice and thought it was not in the commit. Surprised to see it again. No clue what step generated it. Build steps are above my pay grade still.

**Comment by valadas** [comment]
@dnnsoftware/approvers the other PR is merged, my concerns are handled. Thanks @david-poindexter for merging that one :)

File: Dnn.AdminExperience/Dnn.PersonaBar.Extensions/admin/personaBar/Dnn.SiteImportExport/scripts/bundles/siteimportexport-bundle.js.LICENSE.txt (hunk 6 above)
Comment (valadas) [reply Dnn.AdminExperience/Dnn.PersonaBar.Extensions/admin/personaBar/Dnn.SiteImportExport/scripts/bundles/siteimportexport-bundle.js.LICENSE.txt:5]: Yeah, no big deal, it is handled by CI/Build to generate those license files.

**Review by bdukes — APPROVED** [review]
